tests/test_nveto_recorder.py

```diff
@@ -0,0 +1,75 @@
+import strax
+import straxen
+
+import numpy as np
+import unittest
+
+
+class TestMergeIntervals:
+
+    def test_empty_intervals(self):
+        intervals = np.zeros(0, dtype=strax.time_fields)
+        intervals = straxen.merge_intervals(intervals)
+        assert len(intervals) == 0, 'Empty input should return empty intervals!'
+
+    def test_merge_overlapping_intervals(self):
+        intervals = np.zeros(4, dtype=strax.time_fields)
+
+        # First interval:
+        intervals[0]['time'] = 2
+        intervals[0]['endtime'] = 4
+        intervals[1]['time'] = 3
+        intervals[1]['endtime'] = 7
+        intervals[2]['time'] = 7
+        intervals[2]['endtime'] = 8
+
+        # Second interval:
+        intervals[3]['time'] = 20
+        intervals[3]['endtime'] = 22
```

Comment:
_minor_: I guess you thought this might be useful for the sake of the comments right? I think it might read simpler if you just used `intervals['time'] = [2, 3, 4, 7, 20]` or even `intervals = np.ndarray([[2, 3, 4, 7, 20], ....., dtype=...)`

Especially since you do this for several tests below.